scripts/tigers.coffee

```diff
@@ -11,202 +11,32 @@
-{ date: Date.parse('8/20/16'), desc: "Boston 7:10" },
-{ date: Date.parse('8/21/16'), desc: "Boston 1:10" },
-{ date: Date.parse('8/23/16'), desc: "at Minnesota 8:10" },
-{ date: Date.parse('8/24/16'), desc: "at Minnesota 8:10" },
-{ date: Date.parse('8/25/16'), desc: "at Minnesota 1:10" },
-{ date: Date.parse('8/26/16'), desc: "LA Angels 7:10" },
-{ date: Date.parse('8/27/16'), desc: "LA Angels 7:10" },
-{ date: Date.parse('8/28/16'), desc: "LA Angels 1:10" },
-{ date: Date.parse('8/29/16'), desc: "Chicago Sox 7:10" },
-{ date: Date.parse('8/30/16'), desc: "Chicago Sox 7:10" },
-{ date: Date.parse('8/31/16'), desc: "Chicago Sox 1:10" },
-{ date: Date.parse('9/2/16'), desc: "at Kansas City 8:15" },
-{ date: Date.parse('9/3/16'), desc: "at Kansas City 7:15" },
-{ date: Date.parse('9/4/16'), desc: "at Kansas City 2:15" },
-{ date: Date.parse('9/5/16'), desc: "at Chicago Sox 4:10" },
-{ date: Date.parse('9/6/16'), desc: "at Chicago Sox 8:10" },
-{ date: Date.parse('9/7/16'), desc: "at Chicago Sox 2:10" },
-{ date: Date.parse('9/9/16'), desc: "Baltimore 7:10" },
-{ date: Date.parse('9/10/16'), desc: "Baltimore 7:08" },
-{ date: Date.parse('9/11/16'), desc: "Baltimore 1:10" },
-{ date: Date.parse('9/12/16'), desc: "Minnesota 7:10" },
-{ date: Date.parse('9/13/16'), desc: "Minnesota 7:10" },
-{ date: Date.parse('9/14/16'), desc: "Minnesota 7:10" },
-{ date: Date.parse('9/15/16'), desc: "Minnesota 1:10" },
-{ date: Date.parse('9/16/16'), desc: "at Cleveland 7:10" },
-{ date: Date.parse('9/17/16'), desc: "at Cleveland 4:10" },
-{ date: Date.parse('9/18/16'), desc: "at Cleveland 1:10" },
-{ date: Date.parse('9/20/16'), desc: "at Minnesota 8:10" },
-{ date: Date.parse('9/21/16'), desc: "at Minnesota 8:10" },
-{ date: Date.parse('9/22/16'), desc: "at Minnesota 8:10" },
-{ date: Date.parse('9/23/16'), desc: "Kansas City 7:10" },
-{ date: Date.parse('9/24/16'), desc: "Kansas City 7:10" },
-{ date: Date.parse('9/25/16'), desc: "Kansas City 1:10" },
-{ date: Date.parse('9/26/16'), desc: "Cleveland 7:10" },
-{ date: Date.parse('9/27/16'), desc: "Cleveland 7:10" },
-{ date: Date.parse('9/28/16'), desc: "Cleveland 7:10" },
-{ date: Date.parse('9/29/16'), desc: "Cleveland 1:10" },
-{ date: Date.parse('9/30/16'), desc: "at Atlanta 7:35" },
-{ date: Date.parse('10/1/16'), desc: "at Atlanta 7:10" },
-{ date: Date.parse('10/2/16'), desc: "at Atlanta 3:10" }]
-
-daysOfWeek = [
-  'Sunday'
-  'Monday'
-  'Tuesday'
-  'Wednesday'
-  'Thursday'
-  'Friday'
-  'Saturday'
-]
-
-displayDate = (date, now) ->
-  day = date.getDay()
-  dayOfWeek = daysOfWeek[day]
-  if Date.compare(date, now) == 0
-    dayOfWeek = "Today"
-  else if day == now.getDay() + 1
-    dayOfWeek = "Tomorrow"
-  "#{dayOfWeek} #{date.getMonth() + 1}/#{date.getDate()}"
+formatMessage = (json) ->
+  formatDate = (date) -> moment(date, 'YYYY-MM-DD').format('ddd MMM DD')
+  formatTime = (time) -> if time then moment(time).format('h:mma') else 'TBD'
+  formattedGames = json.map (obj) ->
```

Comment:
Does the API control the amount of games returned? Hopefully we can have just this week or so and not the whole schedule.